scripts/storage-emulator-integration/tests.ts

```diff
@@ -1373,87 +1232,44 @@ describe("Storage emulator", () => {
       await page.goto("https://example.com", { waitUntil: "networkidle2" });
 
       await page.addScriptTag({
-        url: "https://www.gstatic.com/firebasejs/7.24.0/firebase-app.js",
+        url: "https://www.gstatic.com/firebasejs/9.9.1/firebase-app-compat.js",
       });
       await page.addScriptTag({
-        url: "https://www.gstatic.com/firebasejs/7.24.0/firebase-auth.js",
+        url: "https://www.gstatic.com/firebasejs/9.9.1/firebase-auth-compat.js",
       });
       await page.addScriptTag({
-        url: TEST_CONFIG.useProductionServers
-          ? "https://www.gstatic.com/firebasejs/7.24.0/firebase-storage.js"
-          : "https://storage.googleapis.com/fir-tools-builds/firebase-storage.js",
+        url: "https://www.gstatic.com/firebasejs/9.9.1/firebase-storage-compat.js",
```

Comment:
Why don't we need "https://storage.googleapis.com/fir-tools-builds/firebase-storage.js" anymore?

Reply:
Not sure why this was here in the first place, it seems like was designed to support testing against custom built js sdk version but this seems undesirable from a testing perspective